CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
 ## UNRELEASED
```

Comment:
Does CRT pick out the right changelog section if the UNRELEASED header is left here?

Reply:
Good call. I'll remove it just in case